app/Decsys/Models/Webhooks/PayloadModel.cs

```diff
@@ -0,0 +1,21 @@
+namespace Decsys.Models.Webhooks;
+
+public class PayloadModel
+{ 
+    public PayloadModel(int surveyId, int instanceId, string participantId, BaseEventType eventType, PageResponseSummary? pageResponseSummary)
+    {
+        SurveyId = surveyId;
+        InstanceId = instanceId;
+        EventType = eventType;
+        PageResponseSummary = pageResponseSummary;
+        ParticipantId = participantId;
+    }
+
+    public DateTimeOffset Timestamp { get; set; } = DateTimeOffset.UtcNow;
+    public string ParticipantId { get; set; }
+    public int SurveyId { get; set; }
+    public int InstanceId { get; set; }
+    public BaseEventType EventType { get; set; }
+    public PageResponseSummary? PageResponseSummary { get; set; } 
```

Comment:
You've got a minor problem here, but I suggest we leave it alone for the time being.

PageResponseSummary is an appropriate data payload for a PAGE_NAVIGATION event, but not for all event types.

So like filter criteria, we also need to segregate typed data payloads.

However, since we're only implementing PAGE_NAVIGATION currently, and this is early stage implementation, I'm happy to fix this further down the line, at least beyond this PR ;)

Reply:
Yep definitely focussed on the page navigation rather than build out further types. 
Maybe it should just be dynamic - but for further discussion.